src/analyses/basePriv.ml

```diff
@@ -348,6 +352,188 @@ struct
       st
 end
 
+module PerMutexMeetTIDPriv: S =
+struct
+  include PerMutexPrivBase
+  include PerMutexTidG(struct
+      let exclude_not_started () = GobConfig.get_bool "ana.apron.priv.not-started"
+      let exclude_must_joined () = GobConfig.get_bool "ana.apron.priv.must-joined"
```

Comment:
This base privatization still refers to Apron options. I don't think it's worth having separate options for these, rather maybe move them up like `ana.priv.not-started`.

Reply:
I duplicated them for now so we still have fine-grained control while we are working on the more-traces paper. Once that is done, I think we can pull things up as you suggested.